### PR TITLE
Update doc about site_credentials

### PR DESCRIPTION
**Pull request by j0k3r**
See https://github.com/wallabag/wallabag/pull/2683

The doc now use the [gitbook-plugin-alerts](https://github.com/hypertrack/gitbook-plugin-alerts) to configure kind of information / urgent section.

### CODE REVIEW

File: book.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
       "page-toc",
       "-sharing",
       "hints",
-      "versions"
+      "versions",
+      "alerts"
```
Comment (tcitworld) [inline book.json]: We already had the `hints` plugin that did the job, but this one seems better. Changes needed therefore.

File: en/user/articles/restricted.md [hunk 2]
```diff
@@ -17,3 +17,31 @@ In version 2.2, only the administrator could put his credentials in a config fil
...
+
+Click on it to go to the management of your site credentials. You'll be able to add many login / password.
+
+> **[info] Information**
+>
+> These information will only be accessible by **YOU** and no other users on the wallabag instance.
+
+## Security
+
+Login and password you'll set will be encrypted in the database which means the database administrator (and/or the admin of your wallabag instance) won't be able to read your credentials.
```
Comment (tcitworld) [inline en/user/articles/restricted.md]: > However, the server administrator will have access.

File: book.json (hunk 1 above)
Comment (j0k3r) [inline book.json]: I can't "hints" in the doc, could you tell me where it is used?

Comment (tcitworld) [inline book.json]: https://github.com/wallabag/doc/commit/e16f49012b7945c902ea4bceac994954b85c88bb

Comment (tcitworld) [inline book.json]: But there may be more

**Pull request merged**